contracts/contracts/eosio.system/src/delegate_bandwidth.cpp

```diff
@@ -28,6 +29,7 @@ namespace eosiosystem {
 
    static constexpr uint32_t refund_delay_sec = 3*24*3600;
    static constexpr int64_t  ram_gift_bytes = 1400;
+   const eosio::time_point two_month = eosio::time_point(eosio::seconds(5097600)); // two month
```

Comment:
@AndreiAndrusyshyn I'm not yet sure about best way for non-constexpr constants but it seems better to keep it static and initialize in cpp;

as well you can use eosio::days it's more readable for human

two_month is unused var ( I suppose it should be 6month); anyway its to specific name, something like resignation_period/stake_lock_period is better